Parsers/ASimWebSession/Parsers/ASimWebSessionCiscoMeraki.yaml

```diff
@@ -0,0 +1,228 @@
+              replace_string(tostring(action), "'", "")
+          ),
+          agent= iff(
+            agent has '"',
+            replace_string(tostring(agent), '"', ''),
+            replace_string(tostring(agent), "'", "")
+        ),
+          sha256= iff(
+              sha256 has '"',
+              replace_string(tostring(sha256), '"', ''),
+              replace_string(tostring(sha256), "'", "")
+          ),
+          fsha256= iff(
+              sha256 has '"',
+              replace_string(tostring(fsha256), '"', ''),
+              replace_string(tostring(fsha256), "'", "")
+          ),
+          name= iff(
+            name has '"',
+            replace_string(tostring(name), '"', ''),
+            replace_string(tostring(name), "'", "")
+        ),
+          mac= iff(
+          mac has '"',
+          replace_string(tostring(mac), '"', ''),
+          replace_string(tostring(mac), "'", "")
+      ),
+          url= iff(
+          url has '"',
+          replace_string(tostring(url), '"', ''),
+          replace_string(tostring(url), "'", "")
+      ),
+          urls= iff(
+            urls has '"',
+            replace_string(tostring(urls), '"', ''),
+            replace_string(tostring(urls), "'", "")
+        )
+      | lookup ActionLookup on action
+      | extend Url = coalesce(url, urls)
+      | extend
+          EventResult=case(
+                  action in~("allow", "log", "accept") or LogType == "urls",
+                  "Success",
+                  action in~("block", "deny", "quarantine"),
+                  "Failure",
+                  ""
+              ),
+          EventSeverity=case(
+                    action in~("block", "deny", "quarantine") and disposition == "malicious",
+                    "Medium",
+                    action in~("allow", "log", "accept") and disposition == "malicious",
+                    "High",
+                    action in~("block", "deny", "quarantine") and disposition != "malicious",
+                    "Low",
+                    "Informational"
+                ),
+          Ip=extract("(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))", 1, url)
+      | extend SrcIpAddr = iff(
+                          src has ".",
+                          split(src, ":")[0], 
+                          coalesce(temp_srcip, src)
+                      )
+      | extend SrcPortNumber = toint(
+                              iff (
+      src has ".",
+      split(src, ":")[1],
+      column_ifexists("temp_srcport", " ")
+  )
+                          )
+      | extend DstIpAddr = iff(
+                          dst has ".",
+                          split(dst, ":")[0], 
+                          coalesce(temp_dstip, dst)
+                      )
+      | extend DstPortNumber = toint(
+                              iff (
+      dst has ".",
+      split(dst, ":")[1],
+      column_ifexists("temp_dstport", " ")
```

Comment:
need for temp_dstport?

Reply:
We have extracted the raw values from dst field and then we are processing it to extract port and ip from it in case of ipv6 address. So temp_dstport will contain value of port in case of ipv6 address.